core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -534,5 +534,25 @@ class RemoteServicesMockServer(port:Int) extends RawlsTestUtils {
       )
   }
 
+  mockServer.when(
+    request()
+      .withMethod("GET")
+      .withPath("/workflows/v1/29b2e816-ecaf-11e6-b006-92361f002671/outputs")
```

Comment:
hmm. this'd require some refactoring to Do It Right. i'll put in a tech debt ticket